tube-player/modules/01-Getting-started/README.md

```diff
@@ -126,11 +126,9 @@ As explained [in the intro](xref:Workshop.TubePlayer.Overview#tube-player-worksh
 
 ## Running the application
 
-1. Ensure a startup project is selected. Either right-click the desired project head in Solution Explorer (e.g. *TubePlayer.Mobile*) and select *Set as startup project*, or select this project from the Startup projects dropdown.
-    When selecting *Mobile*, you could then select which mobile platform to run and on which emulator.  
-    The emulator can be selected from the subsequent menu as shown in the picture:
+1. Select the target framework (e.g., Android, iOS, Windows) and the device or emulator you want to deploy to. The emulator can be selected from the subsequent menu as shown in the picture:
```

Comment:
```suggestion
1. Select the target framework (e.g., Android, iOS, Windows) and the device or emulator you want to deploy. The emulator can be selected from the subsequent menu as shown in the picture:
```

Reply:
The word "to" is needed in that sentence.